ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/services/UserManagementServiceAuthorization.scala

```diff
@@ -29,39 +29,60 @@ private[daml] final class UserManagementServiceAuthorization(
     new DamlContextualizedErrorLogger(logger, loggingContext, None)
 
   override def createUser(request: CreateUserRequest): Future[CreateUserResponse] =
-    authorizer.requireAdminClaims(service.createUser)(request)
+    request.user match {
+      case Some(user) =>
+        authorizer.requireAdminIDPContext(user.identityProviderId, service.createUser)(request)
+      case None =>
+        authorizer.requireAdminOrIDPAdminClaims(service.createUser)(request)
```

Comment:
Could we treat `user = None` the same as `user.identityProviderId = ""`?
That would aslo match grpc's default value semantics

Reply:
We discussed this with @mziolekda and agreed that we do not have this convenience for `identity_provider_id` argument.